CoZ Application Form.md

```diff
@@ -1,25 +1,25 @@
 # City of Zion Application Process
-Updated 09/21/17
+*Updated 09/26/17*
 
-The City of Zion (CoZ) is a group of open source developers formed for the purpose of furthering the progress of the NEO blockchain. We are always looking for talented people to join our ranks. 
+The **City of Zion** (**CoZ**) is a group of open source developers formed for the purpose of furthering the progress of the NEO blockchain. We are always looking for talented people to join our ranks. 
 
-If you want to help with one of our projects, you can do it directly through GitHub (see the Community Workflow). Our repository is located at https://github.com/CityOfZion. You can also join us on project specific Slack channels, access to which will be granted through an application process. Please see the form below.
+If you want to help with one of our projects, you can do it directly through GitHub (see the [Community Workflow](https://github.com/jhepkema/awesome-neo/blob/master/Community%20Workflow.md)). Our repository is located at https://github.com/CityOfZion. You can also join us on project specific Slack channels, access to which will be granted through an application process. Please see the form below.
```

Comment:
The **Community Workflow** has been removed from the repo.

See #12 and #10.

Can you amend the documentation?